packages/firestore/.eslintrc.json

```diff
@@ -2,5 +2,28 @@
     "extends": "../../config/.eslintrc.json",
     "parserOptions": {
         "project": "tsconfig.json"
+    },
+    "plugins": [
+        "import"
+    ],
+    "rules": {
+        "no-console":[ "error", { "allow": ["warn", "error"] }],
+        "import/no-default-export": "error",
+        "@typescript-eslint/no-unused-vars": [
+            "error",
+            {
+                "varsIgnorePattern": "^_",
+                "args": "none"
+            }
+        ]
+    },
+    "overrides": {
+        "files": [
+            "*test.ts",
+            "**/test/**/*.ts"
+        ],
+        "rules": {
+            "no-console": "off"
```

Comment:
why did you disable this?  People often add console.log() statements while debugging tests and then accidentally forget to remove them.